packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/test.sh

```diff
@@ -16,6 +16,47 @@ test_val "$(compare_swing_store_export_data $EXPORT_DIR)" "match" "swing-store c
 rm -rf $EXPORT_DIR
 startAgd
 
+testHasChildren() {
```

Comment:
Clarifying suggestion:
```suggestion
testChildCountGreaterThan() {
```
or even (with refactoring from `>` to `>=`):
```suggestion
testMinChildren() {
```